### ISSUE TITLE
Icon/logo suggestion

### ISSUE THREAD

**Issue by RUvlad1** [uowuo/abaddon#263]
I've been thinking about making some kind of icon or logo for Abaddon, since it looks a bit awkward without one, and I think I managed to come up with something

![image](https://github.com/uowuo/abaddon/assets/60317461/73df7253-c7d8-49d7-b232-bc32d7f4e3ea)

The wordmark isn't quite as interesting, it's basically just modified Rubik

![image](https://github.com/uowuo/abaddon/assets/60317461/a118b55c-e5c0-444b-a698-99a8b8b9ccb4)

I'm not sure if I'm done wit this yet, if you have any suggestions, please let me know!
I'll send a proper png of the icon, as well as a svg export once I'm fully finished with it

**Comment by ouwou** [comment]
it does need an icon. i was gonna try making something in a week or two. i was feeling something more of a character (like discord has clyde) as opposed to something more generic like a chat bubble 😼

**Comment by ouwou** [comment]
yeah. i dont wanna rip it off i just mean itd be cool if abaddon had its own little mascot xd

**Comment by RUvlad1** [comment]
> yeah. i dont wanna rip it off i just mean itd be cool if abaddon had its own little mascot xd

forgot to reply but idk if i can do something like that, i dont really have any ideas

idk if there's anything else i can add but here's the svg and stuff

![icon](https://github.com/uowuo/abaddon/assets/60317461/b9cb5c2d-0896-4edb-b442-2d2751dccd27)
![icon](https://github.com/uowuo/abaddon/assets/60317461/0b96c4e4-e734-47c7-96be-b1177b58acee)
<img width="400" alt="icon@2x" src="https://github.com/uowuo/abaddon/assets/60317461/9f19de27-fd86-4913-ad85-882c1712f052">
![wordmark](https://github.com/uowuo/abaddon/assets/60317461/44e6d1f2-4a1b-4ed6-91f4-195161c56bf1)
![wordmark](https://github.com/uowuo/abaddon/assets/60317461/801bdeba-60a7-4e92-bdda-292550f37ba2)
<img width="1000" alt="wordmark@2x" src="https://github.com/uowuo/abaddon/assets/60317461/be091760-af6f-4793-9c6a-03edbb54ae8b">

**Comment by iAmInActions** [comment]
I randomly happened to stumble upon this thread again and thought, what about a cat holding a box, letter or maybe a phone?

here is some quick and dirty concept art of how this could look, this is defo not good enough for an icon though.
![cat](https://github.com/uowuo/abaddon/assets/83808704/2707dfd2-1db0-44e3-a227-5cac577e26c4)

**Comment by ouwou** [comment]
> I randomly happened to stumble upon this thread again and thought, what about a cat holding a box, letter or maybe a phone?

lol i like the idea :3 box is probably my favorite

> So, any other thoughts on this? Would be nice if this was used, ateast the current version that I came up with

missed this but i dont wanna use anything if its just gonna be temporary. ive claimed the gtk box for that purpose 😹

**Comment by iAmInActions** [comment]
So, i just remembered this issue and thought I'd drop a quick icon concept art thingie of the "abaddon cat" here :3
![Image](https://github.com/user-attachments/assets/fd6ac9e8-6bfc-44e1-af02-620d4860c23d)

**Comment by 9thIvy** [comment]
@iAmInActions Mind if I trace it and convert it into an SVG icon for the flatpak port?

**Comment by iAmInActions** [comment]
> [@iAmInActions](https://github.com/iAmInActions) Mind if I trace it and convert it into an SVG icon for the flatpak port?

would appreciate an SVG version ngl, having it scaleable is certainly an advantage. just make sure to share it here, would like to add it to the ppc xbps package in my repo too

**Comment by 9thIvy** [comment]
> > [@iAmInActions](https://github.com/iAmInActions) Mind if I trace it and convert it into an SVG icon for the flatpak port?
> 
> would appreciate an SVG version ngl, having it scaleable is certainly an advantage. just make sure to share it here, would like to add it to the ppc xbps package in my repo too

I have my poor traced interpretation with no legs because they looked cursed when I drew them, and then a machine converted version below
![Image](https://github.com/user-attachments/assets/ce29449b-0b0c-4313-bdc2-058b9c43fa4b)
![Image](https://github.com/user-attachments/assets/892f6c37-57c5-4230-b601-c9c3d36ecef1)

**Comment by iAmInActions** [comment]
> > > [@iAmInActions](https://github.com/iAmInActions) Mind if I trace it and convert it into an SVG icon for the flatpak port?
> > 
> > 
> > would appreciate an SVG version ngl, having it scaleable is certainly an advantage. just make sure to share it here, would like to add it to the ppc xbps package in my repo too
> 
> I have my poor traced interpretation with no legs because they looked cursed when I drew them, and then a machine converted version below

I did a few adjustments to the traced version to make it look a bit closer to the initial sketch i had made, please let me know if you prefer this version.

![Image](https://github.com/user-attachments/assets/5b79cb38-ba09-4035-92a0-7178711df62e)

**Comment by doctorsangria** [comment]
> So, i just remembered this issue and thought I'd drop a quick icon concept art thingie of the "abaddon cat" here :3 ![Image](https://github.com/user-attachments/assets/fd6ac9e8-6bfc-44e1-af02-620d4860c23d)

Might just be me but I'm really digging the rasterized mspaint look over a vector, it suits the handmade drawing a lot better imo (and its just so cozy :3)

Could also be a more faithful representation of the project itself (home-made passion project rather than a sleek, corporate product). Scaling it nicely won't be as simple ofc but it could work with some effort, I give it a shot myself with GIMP when I have some extra time. Though again thats just me
